pymbar/bar.py

```diff
@@ -289,10 +297,8 @@ def BAR(w_F, w_R, DeltaF=0.0, compute_uncertainty=True, uncertainty_method='BAR'
             # The free energy difference appears to be zero -- return.
             if verbose:
                 print('The free energy difference appears to be zero.')
-            if compute_uncertainty:
-                return [0.0, 0.0]
-            else:
-                return 0.0
+            result_vals['Delta_f'] = 0.0 
+            break
```

Comment:
Did you mean to change this to a `break` instead of a return?

Reply:
No, should be break, so that it can then compute the uncertainty.  But actually, it shouldn't set results there, it should just break, since DeltaF gets copied to results_vals anyway.